build.gradle.kts

```diff
@@ -271,3 +287,10 @@ tasks {
 		newTagRevision = System.getenv("GITHUB_SHA")
 	}
 }
+
+tasks.withType<Javadoc>().configureEach {
```

Comment:
Ideally this would be part of the `javadoc` task definition above, but then it failed to find functions, possibly because the 'wrong' `this` was in scope.
Would be great if someone more familiar with the Gradle Kotlin DSL could give it a try.

Alternatively the complete `javadoc` block above could be put inside `tasks.javadoc { ... }` including the toolchain definition below, but then the tasks configurations are separated which might decrease readability.

Reply:
Thanks, that worked!